public/locales/en/common.json

```diff
@@ -44,10 +47,24 @@
   },
   "newcomer" : "New to MAPLE? For extra support, check",
   "notInCommittee": "Bill not currently in committee",
+  "noResults": "No Results",
   "orgs": "Organizations",
+  "partnerLogos":{
+    "NuLaw": "Northeastern School of Law Logo",
+    "CodeForBoston": "Code for Boston Logo",
+    "OpenCollective":"Open Collective Logo"
```

Comment:
nit: Since this PR opened, we moved to Partners in Democracy - we may want to cover that as well.